spec/controllers/stafftools/assignment_repos_controller_spec.rb

```diff
@@ -4,7 +4,12 @@
   let(:user)    { GitHubFactory.create_owner_classroom_org.users.first }
   let(:student) { GitHubFactory.create_classroom_student               }
 
-  let(:assignment) { create(:assignment, creator: user, organization: user.organizations.first, public_repo: false) }
```

Comment:
Strange things happened when I was recording this test case. It's like every time I run this test the `assignment` here is a different one having a different name and I have to record a new VCR each time, so I changed it into a static one. However I don't know if it's fine.